File: BACKEND/BE/Models/Comentario.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BE.Models
{
    public class Comentario
    {
        public int ID { get; set; } 
        public DateTime FechaHora { get; set; }

        public string Contenido { get; set; } = null!;

        public virtual Usuario Corredor { get; set; }


        [ForeignKey("Corredor")]
        public int CorredorID { get; set; } 


        public virtual Evento Evento { get; set; }


        [ForeignKey("Evento")]
        public int EventoID { get; set; }
        
       



    }
}

File: FRONTEND/FE-CRUDEventos/src/app/interfaces/comentario.ts
export interface Comentario {
  id: number;
  fechaHora: Date;
  contenido: string;
  nombreCorredor: string;
  corredorID: number;
  imagenCorredor: string;
}
export interface ComentarioNuevo {
  fechaHora: string;
  contenido: string;
  corredorID: number;
  nombreCorredor: string;
  eventoID: number;
}
